.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish Package to npmjs
+on:
+  push:
+    tags:
+      - v**
+      - test-ci-**
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
```

Comment:
Recommend adding `cache: 'npm'` which afaik should cache packages between builds.

(This applies to the other workflow too)

Reply:
done